Let's merge the two branches and work from yours moving forward

tokyojeffro opened the pull request:
Haven't used this yet, should be interesting...

ThurstonC committed:
Create README.md

ThurstonC committed:
Update BayItem.java

Backed out one change in a step 3 condition to try to restore operation (line 145)

ThurstonC committed:
JTC rewrite of bayitem

May not have really incorporated Jeff test 3 changes properly

ThurstonC committed:
JTC rewrite of bayitem version JTC1.0

Commit:
version number put in gradsle build file (2)

Commit:
Crittercism added back in (put .jar in the correct place).

tokyojeffro commented:
Merging JTC's changes with mine.

ThurstonC commented:
Hi Jeff,  great to hear from you.   We should talk some to catch up, it's
been since the holidays.

Fortunately my last update on the test rack is still accurate, they've been
using it successfully as the issues that come up all have reasonably quick
workarounds.

Take care,
Thurston
On Jan 31, 2016 10:09 PM, "Jeff Griffin" notifications@github.com wrote:

> ## Haven't used this yet, should be interesting...
> 
> You can view, comment on, or merge this pull request online at:
> 
>   https://github.com/tokyojeffro/ScentWave/pull/1
> Commit Summary
> - Create README.md
> - Update BayItem.java
> - JTC rewrite of bayitem
> - JTC rewrite of bayitem version JTC1.0
> - version number put in gradsle build file (2)
> - Crittercism added back in (put .jar in the correct place).
> 
> File Changes
> - _M_ .idea/gradle.xml
>   https://github.com/tokyojeffro/ScentWave/pull/1/files#diff-0 (4)
> - _M_ .idea/modules.xml
>   https://github.com/tokyojeffro/ScentWave/pull/1/files#diff-1 (5)
> - _A_ README.md
>   https://github.com/tokyojeffro/ScentWave/pull/1/files#diff-2 (3)
> - _D_ ScentWave.iml
>   https://github.com/tokyojeffro/ScentWave/pull/1/files#diff-3 (19)
> - _A_ ScentWaveJTC.iml
>   https://github.com/tokyojeffro/ScentWave/pull/1/files#diff-4 (19)
> - _M_ app/app.iml
>   https://github.com/tokyojeffro/ScentWave/pull/1/files#diff-5 (116)
> - _M_ app/build.gradle
>   https://github.com/tokyojeffro/ScentWave/pull/1/files#diff-6 (4)
> - _M_ app/src/main/AndroidManifest.xml
>   https://github.com/tokyojeffro/ScentWave/pull/1/files#diff-7 (11)
> - _M_ app/src/main/java/com/scentair/scentwave/BayItem.java
>   https://github.com/tokyojeffro/ScentWave/pull/1/files#diff-8 (361)
> 
> Patch Links:
> - https://github.com/tokyojeffro/ScentWave/pull/1.patch
> - https://github.com/tokyojeffro/ScentWave/pull/1.diff
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/tokyojeffro/ScentWave/pull/1.

tokyojeffro commented:
So we don't need new code?
On Feb 1, 2016 18:59, "Thurston" notifications@github.com wrote:

> Hi Jeff, great to hear from you. We should talk some to catch up, it's
> been since the holidays.
> 
> Fortunately my last update on the test rack is still accurate, they've been
> using it successfully as the issues that come up all have reasonably quick
> workarounds.
> 
> Take care,
> Thurston
> On Jan 31, 2016 10:09 PM, "Jeff Griffin" notifications@github.com wrote:
> 
> > ## Haven't used this yet, should be interesting...
> > 
> > You can view, comment on, or merge this pull request online at:
> > 
> > https://github.com/tokyojeffro/ScentWave/pull/1
> > Commit Summary
> > - Create README.md
> > - Update BayItem.java
> > - JTC rewrite of bayitem
> > - JTC rewrite of bayitem version JTC1.0
> > - version number put in gradsle build file (2)
> > - Crittercism added back in (put .jar in the correct place).
> > 
> > File Changes
> > - _M_ .idea/gradle.xml
> >   https://github.com/tokyojeffro/ScentWave/pull/1/files#diff-0 (4)
> > - _M_ .idea/modules.xml
> >   https://github.com/tokyojeffro/ScentWave/pull/1/files#diff-1 (5)
> > - _A_ README.md
> >   https://github.com/tokyojeffro/ScentWave/pull/1/files#diff-2 (3)
> > - _D_ ScentWave.iml
> >   https://github.com/tokyojeffro/ScentWave/pull/1/files#diff-3 (19)
> > - _A_ ScentWaveJTC.iml
> >   https://github.com/tokyojeffro/ScentWave/pull/1/files#diff-4 (19)
> > - _M_ app/app.iml
> >   https://github.com/tokyojeffro/ScentWave/pull/1/files#diff-5 (116)
> > - _M_ app/build.gradle
> >   https://github.com/tokyojeffro/ScentWave/pull/1/files#diff-6 (4)
> > - _M_ app/src/main/AndroidManifest.xml
> >   https://github.com/tokyojeffro/ScentWave/pull/1/files#diff-7 (11)
> > - _M_ app/src/main/java/com/scentair/scentwave/BayItem.java
> >   https://github.com/tokyojeffro/ScentWave/pull/1/files#diff-8 (361)
> > 
> > Patch Links:
> > - https://github.com/tokyojeffro/ScentWave/pull/1.patch
> > - https://github.com/tokyojeffro/ScentWave/pull/1.diff
> > 
> > —
> > Reply to this email directly or view it on GitHub
> > https://github.com/tokyojeffro/ScentWave/pull/1.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/tokyojeffro/ScentWave/pull/1#issuecomment-178265802.

ThurstonC commented:
Hi Jeff, We do still need the one fix to the crash at end of test.  Beyond
that if you've thought up a way to auto recover the driver issues it would
be great to try, and save some time, but it's not a must have since
rebooting the server works.
  I'll check out your new build tomorrow.

Have a good week!
Thurston

Ps:  Our new EE is already leaving to do real circuit design somewhere.
Where were these jobs when I looked...  he's certainly a good contact for
me in any case.
On Feb 1, 2016 7:22 PM, "Jeff Griffin" notifications@github.com wrote:

> So we don't need new code?
> On Feb 1, 2016 18:59, "Thurston" notifications@github.com wrote:
> 
> > Hi Jeff, great to hear from you. We should talk some to catch up, it's
> > been since the holidays.
> > 
> > Fortunately my last update on the test rack is still accurate, they've
> > been
> > using it successfully as the issues that come up all have reasonably
> > quick
> > workarounds.
> > 
> > Take care,
> > Thurston
> > On Jan 31, 2016 10:09 PM, "Jeff Griffin" notifications@github.com
> > wrote:
> > 
> > > ## Haven't used this yet, should be interesting...
> > > 
> > > You can view, comment on, or merge this pull request online at:
> > > 
> > > https://github.com/tokyojeffro/ScentWave/pull/1
> > > Commit Summary
> > > - Create README.md
> > > - Update BayItem.java
> > > - JTC rewrite of bayitem
> > > - JTC rewrite of bayitem version JTC1.0
> > > - version number put in gradsle build file (2)
> > > - Crittercism added back in (put .jar in the correct place).
> > > 
> > > File Changes
> > > - _M_ .idea/gradle.xml
> > >   https://github.com/tokyojeffro/ScentWave/pull/1/files#diff-0 (4)
> > > - _M_ .idea/modules.xml
> > >   https://github.com/tokyojeffro/ScentWave/pull/1/files#diff-1 (5)
> > > - _A_ README.md
> > >   https://github.com/tokyojeffro/ScentWave/pull/1/files#diff-2 (3)
> > > - _D_ ScentWave.iml
> > >   https://github.com/tokyojeffro/ScentWave/pull/1/files#diff-3 (19)
> > > - _A_ ScentWaveJTC.iml
> > >   https://github.com/tokyojeffro/ScentWave/pull/1/files#diff-4 (19)
> > > - _M_ app/app.iml
> > >   https://github.com/tokyojeffro/ScentWave/pull/1/files#diff-5 (116)
> > > - _M_ app/build.gradle
> > >   https://github.com/tokyojeffro/ScentWave/pull/1/files#diff-6 (4)
> > > - _M_ app/src/main/AndroidManifest.xml
> > >   https://github.com/tokyojeffro/ScentWave/pull/1/files#diff-7 (11)
> > > - _M_ app/src/main/java/com/scentair/scentwave/BayItem.java
> > >   https://github.com/tokyojeffro/ScentWave/pull/1/files#diff-8 (361)
> > > 
> > > Patch Links:
> > > - https://github.com/tokyojeffro/ScentWave/pull/1.patch
> > > - https://github.com/tokyojeffro/ScentWave/pull/1.diff
> > > 
> > > —
> > > Reply to this email directly or view it on GitHub
> > > https://github.com/tokyojeffro/ScentWave/pull/1.
> > 
> > —
> > Reply to this email directly or view it on GitHub
> > <https://github.com/tokyojeffro/ScentWave/pull/1#issuecomment-178265802
> > .
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/tokyojeffro/ScentWave/pull/1#issuecomment-178271003.

ThurstonC commented:
Hi Jeff, it looks like you need to use github desktop to complete the merge.  I sent email with an archive of the project if you'd rather just do it manually by copying over the changed files.

Commit:
Update version to 2.1
Update barcode scanning routines to allow for new barcode style (doesn't end in .00)
Updated minimum length to be 9 digits (was 12)
